code/datums/components/crafting/_recipes.dm

```diff
@@ -45,6 +45,21 @@
 /datum/crafting_recipe/New()
 	if(!(result in reqs))
 		blacklist += result
+	// These should be excluded from all crafting recipies
+	blacklist += list(
+		/obj/item/screwdriver/cyborg,
+		/obj/item/wrench/cyborg,
+		/obj/item/weldingtool/largetank/cyborg,
+		/obj/item/crowbar/cyborg,
+		/obj/item/wirecutters/cyborg,
+		/obj/item/multitool/cyborg,
+		/obj/item/retractor/augment,
+		/obj/item/hemostat/augment,
+		/obj/item/cautery/augment,
+		/obj/item/surgicaldrill/augment,
+		/obj/item/scalpel/augment,
+		/obj/item/circular_saw/augment,
```

Comment:
small nit, can you alphabetize this? there's a few extensions for vscode that will automatically do this for you. looks great otherwise.

Reply:
Did this, thanks